experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -58,31 +65,24 @@ def fml_client(self, version: Optional[NimbusFeatureVersion] = None) -> FmlClien
             logger.error("Nimbus FML Loader: Failed to get FmlClient.")
             return None
 
-    def _get_inspectors(self, client: FmlClient, feature_id: str) -> FmlFeatureInspector:
-        return client.get_feature_inspector(feature_id)
-
-    @staticmethod
-    def _get_errors(inspector: FmlFeatureInspector, blob: str):
-        return inspector.get_errors(blob)
-
     def get_fml_errors(
         self,
         blob: str,
         feature_id: str,
         version: Optional[NimbusFeatureVersion] = None,
     ):
         """Fetch errors from the FML. This method creates FML clients, which are
-        used by `FmlFeatureInspector`s to fetch errors based on the blob of text and
-        the given feature.
+        used by `FmlFeatureInspector`s to fetch FML errors based on a blob of text and
+        the given feature id.
 
         Returns:
             A list of feature manifest errors.
         """
```

Comment:
Nit: like a commit message, have a one sentance summary and then a longer description. i.e.,

```
        """Fetch errors from the FML.

        This method creates FML clients, which are used by `FmlFeatureInspector`s
        to fetch FML errors based on a blob of text and the given feature id.

        Returns:
            A list of feature manifest errors.
            A list of feature manifest errors.
        """
```